.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    name: Prepare release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      # Build
+      - name: Build the file
+        run: |
+          cd /home/runner/work/lovelace-grocy-chores-card/lovelace-grocy-chores-card
+          npm install
+          npm run build
+
+      # Upload build file to the releas as an asset.
+      - name: Upload zip to release
+        uses: svenstaro/upload-release-action@v1-release
+
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: /home/runner/work/lovelace-grocy-chores-card/lovelace-grocy-chores-card/output/grocy-chores-card.js
+          asset_name: grocy-chores-card-pkg.js
```

Comment:
I'm not certain if we want to change the filename or not. 

We could change this to grocy-chores-card.js, maybe worth an experiment.